com.unity.render-pipelines.high-definition/Editor/PostProcessing/DepthOfFieldEditor.cs

```diff
@@ -120,15 +120,10 @@ void DrawQualitySettings()
                 PropertyField(m_FarSampleCount, Styles.k_FarSampleCount);
                 PropertyField(m_FarMaxBlur, Styles.k_FarMaxBlur);
 
-                if (BeginAdditionalPropertiesScope())
-                {
-                    PropertyField(m_Resolution);
-                    PropertyField(m_HighQualityFiltering);
-                    PropertyField(m_PhysicallyBased);
-                    if (m_PhysicallyBased.value.boolValue == true)
-                        EditorGUILayout.HelpBox(Styles.InfoBox, MessageType.Info);
-                }
-                EndAdditionalPropertiesScope();
+                PropertyField(m_Resolution);
```

Comment:
With this change, the help box will still show up with or without additional properties enabled so we still need a scope here.

Reply:
Will be shown with additional properties, what it won't have is the highlight

Reply:
Yeah but don't we want the background highlight to be consistent?

Reply:
(I haven't seen how it looks like but I would assume that we'd rather have the whole background be highlighted)